packages/build-utils/src/clone-env.ts

```diff
@@ -0,0 +1,28 @@
+import type { Env } from './types';
+
+const { hasOwnProperty } = Object.prototype;
+
+/**
+ * Identical to `Object.assign()`, but ensures that the `PATH` environment
```

Comment:
I realize now that `cloneEnv()` is not identical to `Object.assign()` because the former always returns a clone and the latter also mutates the first arg.

We should keep it that way but the comment should be updated since its not true.

Reply:
Fixed!